app/view-plants/page.tsx

```diff
@@ -92,7 +145,12 @@ export default function Page() {
             ) : (
               <div>
                 {plants.map((plant, key) => (
-                  <PlantCard key={key} plant={plant} canSelect={false} />
+                  <InlineDiv
+                    key={key}
+                    onClick={() => handlePlantCardClick(plant)}
+                  >
+                    <PlantCard key={key} plant={plant} canSelect={false} />
+                  </InlineDiv>
```

Comment:
note: we might be able to remove the outer ternary statement just use inAddMode where necessary, since the list of PlantCards is the same each time.

i.e. `canSelect={inAddMode}`,  etc.

but honestly, the current way might be more readable than splitting up the conditioning on inAddMode, so maybe we can leave it